restructuredtext_lint/lint.py

```diff
@@ -80,9 +75,24 @@ def error_collector(data):
     return errors
 
 
-def lint_file(filepath, encoding=None):
+def lint(content, filepath=None, ignore_sphinx=True):
+    """Lint reStructuredText and return errors
+
+    :param string content: reStructuredText to be linted
+    :param string filepath: Optional path to file, this will be returned as the source
+    :rtype list: List of errors. Each error will contain a line, source (filepath),
+        message (error message), and full message (error message + source lines)
+    """
+    if ignore_sphinx:
+        with sphinx.register_unregister_ignores():
+            return _lint(content, filepath=filepath)
+    else:
+        return _lint(content, filepath=filepath)
```

Comment:
This seems to break compat:

''''
Traceback (most recent call last):
  File "/homes/harlowja/dev/os/restructuredtext-lint/restructuredtext_lint/test/test.py", line 42, in test_raises_on_invalid_rst
    actual_errors = self._lint_file(content, filepath)
  File "/homes/harlowja/dev/os/restructuredtext-lint/restructuredtext_lint/test/test.py", line 28, in _lint_file
    return restructuredtext_lint.lint(*args)
TypeError: lint() takes 1 positional argument but 2 were given
''''

So I'll leave it be.